File: ru/ipo/dces/clientservercommunication/AdjustContestRequest.java
package ru.ipo.dces.clientservercommunication;

/**
 * Created by IntelliJ IDEA.
 * User: Илья
 * Date: 21.05.2008
 * Time: 21:58:03
 */
public class AdjustContestRequest implements InfoFrame {

    public String sessionID;
    public ContestDescription contest;
    public ProblemDescription[] problems;

}

File: trunk/Server/tests/data/AdjustContestRequest.php
<?php
require_once('ContestDescription.php');
class AdjustContestRequest{
    public $sessionID;
    public $contest;
    public $problems;

    function __construct(){
        $this->sessionID = null;
        $this->contest = new ContestDescription();
        $this->problems = null;
    }
};
?>
